funccount.py

```diff
@@ -0,0 +1,30 @@
+import os
+import signal
+import subprocess as sp
+import time
+
+class FUNCCOUNT:
```

Comment:
Why is this class all caps?

Reply:
I was following the fio.py but I renamed to capitalize just the first letter